docs/source/io_formats/tallies.rst

```diff
@@ -364,6 +364,10 @@ attributes/sub-elements:
     The mesh library used to represent an unstructured mesh. This can be either
     "moab" or "libmesh". (For unstructured mesh only.)
 
+  :options:
```

Comment:
Can you link to the relevant MOAB documentation here?

Reply:
I put a link in the Python docstring, as that is what users are more likely to see.

Reply:
very nice!